keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -264,54 +231,9 @@ Potential approach:
 
 Memory would need to be somehow bound with this approach.
 
-#### Rejected Option: Return unpaginated responses to paginated list requests
-
-Both for backward compatibility with older versions of Kubernetes that do not support
-pagination, and for compatibility with api-servers that have the watch cache enabled,
-clients must be able to tolerate unpaginated responses for paginated LIST requests.
-
-The kube-apiserver already switches between serving paginated responses and
-serving unpaginated responses depending on if the watch cache is enabled for the
-types requested.
-
-User cases that have disabled the watch cache will still receive paginated responses.
-
-Use cases where paginated was being used to reduce load on etcd, but are able to
-tolerate the data volume of the list being returned in a single response, should
-not impacted.
-
-The most likely problem with this approach is that, because LIST with
-resourceVersion="" is the only way to paginate data from the api-server when the
-watch-cache is enabled (which is the default), clients that need pagination
-(e.g. to avoid receiving too much data in a single request) will be relying on
-LIST resourceVersion="".
-
-We are not planning to pursue this option.
-
-### What if the watch cache is stale?
-
-This design requires wait for a watch cache to catch up to the needed revision
-for consistent reads. If the cache doesn't catch up within some time limit we
-either fail the request for have a fallback.
-
-If the fallback it to forward consistent reads to etcd, a cascading failure
-is likely to occur if caches become stale and a large number of read requests
-are forwarded to etcd.
-
-Since falling back to etcd won't work, we should fail the requests and rely on
-rate limiting to prevent cascading failure.  I.e. `Retry-After` HTTP header (for
-well behaved clients) and [Priority and Fairness](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
-
-### Ability to Opt-out
-
-<<[UNRESOLVED @deads2k]>>
-How to opt out of this behavior and still get a "normal" quorum read? We'll need this ability for our own debugging if nothing else.
-See https://github.com/kubernetes/enhancements/pull/1404#issuecomment-588433911
-<<[/UNRESOLVED]>>
-
 ### Test Plan
 
-Correctness:
+##### Correctness Tests
```

Comment:
You need to update to the new KEP template.

Reply:
Done.